CONTRIBUTING_CODE.md

```diff
@@ -20,7 +20,7 @@ To modify existing Great Expectations code, you complete the following tasks:
 
 - [Submit a pull request](#submit-a-pull-request)
 
-To discuss your code change before you implement it, join the [Great Expectations Slack community](https://greatexpectations.io/slack) and make your suggestion in the [#contributing](https://greatexpectationstalk.slack.com/archives/CV828B2UX) channel. 
+To discuss your code change before you implement it, join the [Great Expectations Slack community](https://greatexpectations.io/slack) and make your suggestion in the [#contributing](https://greatexpectationstalk.slack.com/archives/CV828B2UX) channel.
```

Comment:
@TrangPham I wonder how this got in -- seems better for a separate (albeit tiny) PR.  Thanks.

Reply:
Maybe due to rebasing/merging. Let me fix that